enterprise/web.jsf20/nbproject/project.properties

```diff
@@ -20,3 +20,5 @@ javac.source=1.8
 release.external/javax.faces-2.3.9.jar=modules/ext/jsf-2_2/javax.faces.jar
 release.external/javax.faces-2.3.9-license.txt=modules/ext/jsf-2_2/license.txt
 spec.version.base=1.57.0
+
+sigtest.public.packages=javax.faces,com.sun.faces
```

Comment:
Strangely, this did not occur to me.

Though I think these definitions `web.jsf*` are there for providing libraries to Ant based projects. So technically they should not provide any public api to any NB modules. Also JSF 1.2 is soon 18 years old. Probably it would be better not to include the jars in NB. 

web.jsf20  hide JSF 2.3 we seem to be built our JSF support on that. 

I'm going to file separate PR-s to remove the JSF 1.2  libraries.